bin/oneoff/fix_broken_google_blockly_callouts.rb

```diff
@@ -16,8 +16,20 @@ def fix_callouts
   else
     puts "THIS IS A FULL RUN"
   end
-  puts "***SPRITE LAB***"
-  process_levels(GamelabJr.where(game_id: 64))
+  puts "***PROCESSING ALL BLOCKLY LEVELS***"
```

Comment:
what are "blockly" levels?

Reply:
These are engineer-owned levels that cannot be edited on levelbuilder. You can see them listed here: https://levelbuilder-studio.code.org/levels?name=&level_type=Blockly&script_id=&owner_id=&commit=Search

They include:
* All Star Wars "game" levels
* All Jigsaw levels
* The oldest Maze/Artist/Play Lab levels

Our other level model classes extend the Blockly class, but the records in the database are still 1:1. Processing all Blockly levels doesn't handle sub-types, nor does Maze handle Karel or Studio handle StarWarsGrid. 

We don't use Blockly levels for Dance/Sprite Lab/Poetry/Minecraft, which is why we haven't had to worry about them yet.

Reply:
thanks!